src/Microsoft.VisualStudio.Editors/DesignerFramework/BaseDesignerLoader.vb

```diff
@@ -676,8 +675,7 @@ Namespace Microsoft.VisualStudio.Editors.DesignerFramework
                 Dim LostFocusProjectItem As EnvDTE.ProjectItem = Nothing
                 Try
                     LostFocusProjectItem = LostFocus.ProjectItem
-                Catch ex As Exception
-                    Common.RethrowIfUnrecoverable(ex)
+                Catch ex As Exception When Common.ReportWithoutCrash(ex, NameOf(m_WindowEvents_WindowActivated), NameOf(BaseDesignerLoader))
```

Comment:
Example for Category 2 (Not sure if telemetry is helpful here, but still logging to err on conservative side).